Doc/library/pathlib.rst

```diff
@@ -133,11 +133,14 @@ we also call *flavours*:
       PureWindowsPath('c:/Program Files')
 
    Spurious slashes and single dots are collapsed, but double dots (``'..'``)
-   are not, since this would change the meaning of a path in the face of
-   symbolic links::
+   and initial double slashes (``'//'``) are not, since this would change the
+   meaning of a path in the face of symbolic links and UNC (network) paths
+   respectively::
```

Comment:
```suggestion
   and leading double slashes (``'//'``) are not, since this would change the
   meaning of a path for various reasons (e.g. symbolic links, UNC paths)::
```